lib/pbench/test/functional/agent/cli/commands/tools/test_clear_tools.py

```diff
@@ -55,6 +55,10 @@ def test_clear_tools_test13(monkeypatch, agent_config, pbench_run, pbench_cfg):
 def test_clear_tools_test65(monkeypatch, agent_config, pbench_run, pbench_cfg):
     """ Remove all tools from group good, leave default alone """
     monkeypatch.setenv("_PBENCH_AGENT_CONFIG", str(pbench_cfg))
+    default_group = pbench_run / "tools-v1-default"
+    fubar_default_group = default_group / "fubar2.example.com"
+    fubar_default_group.mkdir(parents=True)
+
```

Comment:
This is a weird state:  we have a group (`default`) which contains a remote (`fubar2.example.com`) which contains _no tools_....

Reply:
Yeah the purpose was to just show that the default group directory is unmolested. However, I see your point, I should add a tool inside the default directory and make an assertion on its existence to confirm that the default groups is unmolested.